components/properties/src/props.rs

```diff
@@ -1425,6 +1425,60 @@ make_enumerated_property! {
     ule_ty: u8;
 }
 
+/// Property Indic_Conjunct_Break.
+/// See UAX #44:
+/// <https://www.unicode.org/reports/tr44/#Indic_Conjunct_Break>.
+///
+/// # Example
+///
+/// ```
+/// use icu::properties::{CodePointMapData, props::IndicConjunctBreak};
+///
+/// assert_eq!(CodePointMapData::<IndicConjunctBreak>::new().get('a'), IndicConjunctBreak::None);
+/// assert_eq!(CodePointMapData::<IndicConjunctBreak>::new().get('\u{094d}'), IndicConjunctBreak::Linker);
+/// assert_eq!(CodePointMapData::<IndicConjunctBreak>::new().get('\u{0915}'), IndicConjunctBreak::Consonant);
+/// assert_eq!(CodePointMapData::<IndicConjunctBreak>::new().get('\u{0300}'), IndicConjunctBreak::Extend);
+/// ```
+#[doc(hidden)] // draft API in ICU4C
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "datagen", derive(databake::Bake))]
+#[cfg_attr(feature = "datagen", databake(path = icu_properties::props))]
+#[allow(clippy::exhaustive_structs)] // newtype
+#[repr(transparent)]
+pub struct IndicConjunctBreak(pub(crate) u8);
+
+impl IndicConjunctBreak {
+    /// Returns an ICU4C `UIndicConjunctBreak` value.
+    pub const fn to_icu4c_value(self) -> u8 {
+        self.0
+    }
+    /// Constructor from an ICU4C `UIndicConjunctBreak` value.
+    pub const fn from_icu4c_value(value: u8) -> Self {
+        Self(value)
+    }
+}
+
+create_const_array! {
+#[allow(missing_docs)] // These constants don't need individual documentation.
+#[allow(non_upper_case_globals)]
+impl IndicConjunctBreak {
```

Comment:
thought: if these names are unstable, should these constants be private/hidden too?

Reply:
I will change to `#[doc(hidden)] instead of #[allow(missing_docs)]`.

Reply:
Please make a follow-up issue to remove the doc(hidden) once it is no longer draft in ICU4C.

Reply:
https://github.com/unicode-org/icu4x/issues/6419 to remove hidden.